core/src/main/java/greencity/webcontroller/ManagementHabitController.java

```diff
@@ -28,8 +28,8 @@
 @Controller
 @AllArgsConstructor
 @RequestMapping("/management/habits")
-public class ManagementHabitsController {
-    private final HabitService habitService;
+public class ManagementHabitController {
```

Comment:
HabitManagementController and HabitManagementService sounds better

Reply:
Changes done.